LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -40,30 +50,43 @@ public void CreateAccounts(int accountCount)
                     {
                         pendingAccountList.Add(account);
                     }
-                }
-
-                Console.WriteLine("create account {0} \n", account);
+                    instructor.Report();
+                }               
             }
+
+            ReportSummary(actionName: action, count: accountCount);
+            ReportFooter(actionName: action);
         }
 
         public void ActivateAccounts()
         {
+            string action = "activate account";
+            int item = 0;
+            ReportHeader(actionName: action);
             for (int i = 0; i < pendingAccountList.Count; i++)
             {
                 pendingAccountList[i].Status = Status.Active;
                 Console.WriteLine($" activated account is{pendingAccountList[i]}\n");
-            }
-            Console.WriteLine($"count is  {pendingAccountList.Count}");
+            }            
+           
+            ReportSummary(actionName: action, count: item);
```

Comment:
item is always 0, you have to print pendingAccountList.Count, but before you clear it, otherwise it will be 0 too.

Reply:
This is not fixed